test/pass/quotes-never-fail.jams

```diff
@@ -0,0 +1,11 @@
+{
+  es1                       "\/"
+  es2                       "\\"
+  es3                       "\n"
+  es4                       "\r"
+  es5                       "\b"
+  es6                       "\f"
+  es7                       "\t"
+  es8                       "\u0024"
```

Comment:
we want to make sure to do escapes like unicode not js utf16

Reply:
Let me double check

Reply:
bump let me know

Reply:
gonna merge as is but this needs to be examined